cobbler.spec

```diff
@@ -484,7 +504,7 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %files web
 %license COPYING
 %doc AUTHORS.in README.md
-%config(noreplace) %{apache_webconfigdir}/cobbler_web.conf
+%config(noreplace) %{apache_etc}vhosts.d/cobbler_web.conf
```

Comment:
@nodeg Have you double checked this? Is this correct or should it be `config.d`?

Reply:
Not yet. Hopefully this afternoon.

Reply:
In my opinion it should be left in `vhosts.d` since it is a virtual host. We should revert this and have a look at the test cases in our CI to figure out what is exactly wrong.

Reply:
It does not work in `vhost.d` . At least not on a SUSE system.

Reply:
@mcalmer But shouldn't it work? In the `httpd.conf` there is this line: `IncludeOptional /etc/apache2/vhosts.d/*.conf`

Reply:
Does the include happen at the same place as conf.d/*.conf? 
Sorry I cannot say if this works. Try it out. It failed pretty fast.
Btw: cobbler_web.conf was not the problem. Only cobbler.conf is used in Uyuni.

Reply:
Setting this to resolved. It's now back to `%{apache_webconfigdir}` as I can see.